semver-diff/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for semver-diff 2.1
+// Project: https://github.com/sindresorhus/semver-diff
+// Definitions by: Chris Barr <https://github.com/chrismbarr>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/semver-diff
+
+declare namespace SemverDiff {
+    type SemverDiffReturn =  'major' | 'minor' | 'patch' | 'prerelease' | 'build' | null;
+
+    export default function(versionA: string, versionB: string): SemverDiffReturn;
+}
+
+declare module 'semver-diff' {
+    export = SemverDiff.default;
```

Comment:
if I do that how will anything know it by the package name which is `'semver-diff'`?

In the tests file it uses `import semverDiff = require('semver-diff');` and making this change will break it so that it no longer knows what this import is